3_Implementation/documentation/html/search/all_c.js

```diff
@@ -0,0 +1,333 @@
+var searchData=
+[
+  ['unity_483',['Unity',['../unity__internals_8h.html#aad738f665f16eb2336b8bc33f432d0da',1,'unity.c']]],
+  ['unity_484',['UNITY',['../unity_8h.html#ae8e6dc0c42ccacb112eb4b836357f991',1,'unity.h']]],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2015pushkar/C-mini-Project-303459-LTTS/pullRequest?prid=7744834)
```suggestion
  ["unity_484",['UNITY',['../unity_8h.html#ae8e6dc0c42ccacb112eb4b836357f991',1,'unity.h']]],
```